card/spec/lib/card/reference_spec.rb

```diff
@@ -80,16 +84,22 @@
     watermelon.update_referencers = true
     watermelon.name="grapefruit"
     watermelon.save!
-    expect(lew.reload.content).to eq("likes [[grapefruit]] and [[grapefruit+seeds|seeds]]")
+    result = "likes [[grapefruit]] and [[grapefruit+seeds|seeds]]"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.